server/storage/rethinkdb.go

```diff
@@ -294,7 +324,100 @@ func (rdb RethinkDB) CheckHealth() error {
 	return nil
 }
 
+func (rdb RethinkDB) writeChange(gun string, version int, sha256, category string) error {
+	now := time.Now()
+	ch := Change{
+		CreatedAt: now,
+		GUN:       gun,
+		Version:   version,
+		SHA256:    sha256,
+		Category:  category,
+	}
+	_, err := gorethink.DB(rdb.dbName).Table(ch.TableName()).Insert(
+		ch,
+		gorethink.InsertOpts{
+			Conflict: "error", // default but explicit for clarity of intent
+		},
+	).RunWrite(rdb.sess)
+	return err
+}
+
 // GetChanges is not implemented for RethinkDB
 func (rdb RethinkDB) GetChanges(changeID string, pageSize int, filterName string) ([]Change, error) {
-	return nil, errors.New("Not Implemented")
+	var (
+		lower, upper, bound []interface{}
+		idx                 = "rdb_created_at_id"
+		max                 = []interface{}{gorethink.Now().Sub(blackoutTime), gorethink.MaxVal}
```

Comment:
is it worth introducing `blackoutTime` to rethinkdb when it doesn't seem to be implemented in mysql?

Reply:
Yes, it deals with the eventual consistency of RethinkDB. MySQL doesn't have an eventual consistency issue.

Reply:
from IRL discussion: would be helpful to leave a comment about the compound min/max corresponding to the behavior of `Between` operating on an index